modules/gapi/src/backends/fluid/gfluidutils.hpp

```diff
@@ -86,6 +86,25 @@ using cv::gapi::own::rintd;
         return;                                            \
     }
 
+#define MERGE3_(T, OP, ...)                                \
```

Comment:
Cannot find place where is it used, could you point me out?

Reply:
discussed offline:
to move out repeated conditions into beginning of function to simplify condition calculation complexity in actual type dispatching routing